website/addons/box/templates/log_templates.mako

```diff
@@ -1,27 +1,27 @@
 <script type="text/html" id="box_file_added">
 added file
-<a class="overflow log-file-link" data-bind="click: NodeActions.addonFileRedirect">{{ params.fullPath }}</a> to
+<a class="overflow log-file-link" data-bind="click: NodeActions.addonFileRedirect">{{ params.fullPath || params.path }}</a> to
 Box in
 <a class="log-node-title-link overflow" data-bind="attr: {href: nodeUrl}">{{ nodeTitle }}</a>
 </script>
 
 <script type="text/html" id="box_folder_created">
 created folder
-<span class="overflow log-folder">{{ params.fullPath }}</span> in
+<span class="overflow log-folder">{{ params.path || params.fullPath }}</span> in
```

Comment:
For the other logs, `params.fullPath` takes precedence over `params.path`. Is this supposed to be different?